src/subcommand/torrent.rs

```diff
@@ -14,6 +15,7 @@ mod verify;
   about("Subcommands related to the BitTorrent protocol.")
 )]
 pub(crate) enum Torrent {
+  Announce(announce::Announce),
```

Comment:
I'm not sure the name of the command should be `announce`. To me the connotation of `announce` is that I'm going to get other peers to connect to me, and doesn't so much have the connotation that we're just going to print out the peers that are present. I think `peers` is good, but I'm interested to hear your thoughts.

Reply:
An announce tells the tracker ---heya, I want H; do you know anyone who's ever wanted H?--- & so clients aren't getting a peer list unless they `announce` that they want one.

Also, there's really not enough complexity here, I think, to justify translating the spec's language into userese.

Reply:
I think discoverability is not so great, since it isn't immediately obvious you can get a list of peers from a command called `announce`. However, it does strike me that an `announce` command could be extended with other things, like the ability to set downloaded/left/uploaded bytes, etc, whereas such extensions wouldn't make sense with a `peers` command.

And, agreed, it's a good idea to avoid translating spec language. So let's go with announce.